samples/sampler/stories/default/Drawer.js

```diff
@@ -68,10 +69,10 @@ storiesOf('Agate', module)
 					>
 						<header>
 							<Heading
-								color={select('Heading color', headingValues.colors, StoryOptions)}
-								showLine={boolean('Heading showLine', StoryOptions)}
-								size={select('Heading size', headingValues.sizes, StoryOptions)}
-								spacing={select('Heading spacing', headingValues.spacings, StoryOptions)}
+								color={select('Heading color', headingValues.colors, HeadingConfig, '')}
+								showLine={boolean('Heading showLine', HeadingConfig)}
+								size={select('Heading size', headingValues.sizes, HeadingConfig, '')}
+								spacing={select('Heading spacing', headingValues.spacings, HeadingConfig, '')}
```

Comment:
I think it is too much adding `Heading` as the `Heading` prop knob naming. Those knobs are already grouped in the `Heading` tab. @MikyungKim , how do you think?

![image](https://user-images.githubusercontent.com/4239873/90583014-2403ca80-e20a-11ea-99c4-58c91af4478c.png)

Reply:
it makes sense.